src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -778,6 +778,7 @@ private static void mapWalkPreferences(NodeAdapter c, WalkPreferences.Builder wa
             "Value should be between 0 and 1." + " If the value is set to be 0, safety is ignored."
           )
           .asDouble(dft.safetyFactor())
-      );
+      )
+      .withEscalatorReluctance(c.of("escalatorReluctance").asDouble(dft.escalatorReluctance()));
```

Comment:
Please add a summary and also an example to https://github.com/opentripplanner/OpenTripPlanner/blob/8db191804459bdb027559361b19ef78c60cdadc4/src/test/resources/standalone/config/router-config.json#L1-L0 (which will then show up in the documentation).